src/models/site/useSiteSettings2.js

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from 'react';
+import { get, merge } from 'lodash-es';
+import axios from 'axios';
+import { useQuery } from 'react-query';
+
+import queryKeys from '../../constants/queryKeys';
+
+export default function useSiteSettings() {
+
+  const settingsResult = useQuery(
+    queryKeys.settingsSchema,  // Howard so what does this actually achieve? it seems to be a text string. Is it for debug or something?
```

Comment:
The purpose of a query key is to give us a way to invalidate cache and trigger refreshes from other places in the app. So for example in usePostSettingsAsset on line 27 you can see us using this same string to invalidate the cache after a new image has been added, which causes another GET of the settings config.

Reply:
Also just wanted to mention, for the old implementation there were two query keys since there were two sources of data. Since we are going down to one source of data we should be able to simplify to just one query key